vendor/assets/stylesheets/dvl/core/sidebar.scss

```diff
@@ -56,6 +56,11 @@
   font-size: 0.9rem;
 }
 
+.sidebar_data_value,
+.sidebar_data_details {
+  @include word_break;
```

Comment:
Name of mixin `word_break` should be written in all lowercase letters with hyphens instead of underscores